core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -101,5 +115,7 @@ class ActivationThrottler(consulServer: String, concurrencyLimit: Int)(
                     subject -> (loadbalancerCount - invokerCount)
             }
         }
+        Future.sequence(publishUserConcurrentActivation())
```

Comment:
Minor nit, but should the `Future.sequence` move to the `publishUserConcurrentActivation` function?
`List[Future[Unit]]` doesn't really make sense as a return type.